rtic_v5/bluepill_blinky/.embed.toml

```diff
@@ -0,0 +1,16 @@
+# Refer to https://github.com/probe-rs/cargo-embed/blob/master/src/config/default.toml
+# for the comprehensive list of options
+
+[default.general]
+chip = "STM32F103"
```

Comment:
Does this even work? I thought the chip needs to be more specific than that. (That's the problem with cargo-embed config files, we cannot omit the chip :slightly_smiling_face: I think I'll open a feature request for that.)

Reply:
Oh, I just found out that apparently cargo-embed now supports specifying the chip on the command line! Very nice.

Therefore I'd suggest leaving away the `chip` configuration in the `.embed.toml` file and instead telling the user to run the command as `cargo embed --release --chip <chip-spec>`. That way no config files need to be adjusted!

Reply:
Sorry, I missed these comments. Could you transfer this to a new PR?